examples/functions-as-entry-points/functions_as_entry_points.out

```diff
@@ -0,0 +1,34 @@
+# At the command line, navigate to the directory that contains the `.bal` file and
+# and use the `ballerina run` command with the `--printreturn` option set to print
+# the return value to the standard out stream.
+
+# use the `ballerina run` command to invoke the `main` function with `add` as the
+# string argument for `op`, and `4` and `8` as the integer arguments on which the
```

Comment:
Shall we add the word specify as follows:
"# string argument for `op`, and specify `4` and `8` as the integer arguments on which the"